bayesapi/resources/tabledata.py

```diff
@@ -6,12 +6,20 @@ class TableDataResource(BaseResource):
     def on_get(self, req, resp):
         table_name = self.cfg.table_name
 
+        quoted_table_name = '"{}"'.format(table_name)
+
         query = self.queries.get_full_table(
-            table_name=table_name
+            table_name = quoted_table_name
```

Comment:
Interestingly, I think this was more idiomatic before. [pep8 recommends](https://www.python.org/dev/peps/pep-0008/#other-recommendations):
>  Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value.